app/javascript/packs/application_css.scss

```diff
@@ -10,13 +10,25 @@ footer.primary-footer {
 .admin-navbar {
   .navbar-brand img {
     height: 35px;
-    width: 35px;
   }
   .dropdown-toggle {
     cursor: pointer;
   }
+  // I added this because several elements actually would show up above the navbar
+  z-index: 999;
 }
 
 .full-screen-table-wrapper {
   overflow-x: scroll;
 }
+
+@include media-breakpoint-up(md) {
+  .sticky-admin-nav-md-screens {
```

Comment:
Why didn't you just apply this to `.admin-navbar` without adding an extra class?

Also, looking at the currently applied css from `.sticky-top` in chrome inspector, it looks like the styles are:

```
.sticky-top {
    position: -webkit-sticky;
    position: sticky;
    top: 0;
    z-index: 1020;
}
```

Is this what you see? If so, you should copy bootstrap 😉 

And now I understand why you were added zindex above! You should instead copy what bootstrap is doing (and use the appropriate variable instead of the number for zindex)